src/org/labkey/test/tests/SecurityTest.java

```diff
@@ -553,6 +555,23 @@ protected void clonePermissionsTest()
         assertNavTrail("Site Users", "User Details", "Permissions");
     }
 
+    protected void projectUsersTest()
+    {
+        beginAt("/project/SecurityVerifyProject/begin.view?");
```

Comment:
Use helper methods when able. It makes future refactors easier.
```suggestion
        beginAt(WebTestHelper.buildUrl("project", getProjectName(), "begin"));
```